TKGame/Player.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.VisualBasic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+
+namespace TKGame
+{
+    class Player : Entity
+    {
+        private static Player instance;
+        public static Player Instance
+        {
+            get
+            {
+                // Creates the player if it doesn't already exist
+                if (instance == null)
+                    instance = new Player();
+
+                return instance;
+            }
+        }
+
+        private Player()
+        {
+            image = Art.Player;
+            // Figure out how to not hard code for now
+            // Starts at (1560, 450) at the middle on the floor level
+            Position = new Vector2(1600/2, 900 - 40);
+        }
+
+        public override void Update(GameTime gameTime)
```

Comment:
This felt fine when I tested it, so I'm not gonna try and think too hard about this lol

Reply:
Just remember, you told me to do it